parallel-install/example/example.go

```diff
@@ -1,9 +1,9 @@
 package main
 
 import (
-	"context"
 	"flag"
 	"fmt"
+	log "github.com/sirupsen/logrus"
```

Comment:
Do we really need this dependency or can we just use golang's `log` package?
Seems a bit too much for just a small example and a unit test.

Reply:
That's a good point. I will remove it.

Reply:
Just use the log package from the go standard lib:
```go
import "log"
```

No need to change anything on the code.

Reply:
I agree but I noticed  that in this file we already use logger from our internal package `logger` `(pkg\logger\logger.go)`
It is instantiated in line 31.  

`log = logger.NewLogger(*verbose)`

Please check the file now. Now it we have only one logger used in this file.